Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM praekeltfoundation/supervisor:alpine
 
 # Install runtime dependencies for MC2 as well as Nginx and Redis
 RUN apk --no-cache add libffi libpq \
-    nginx redis
+    nginx redis; \
```

Comment:
Use `&&`, not `;`. Without `set -e`, `;` means that the rm will still execute and the Docker build will still succeed even if the `apk` command fails. #justshellthings

Alternatively, just do the `rm` in a separate `RUN` command.